### ISSUE TITLE
how to setup pinecone for ingesting data? getting this error

### ISSUE THREAD

**Issue by vvonchain** [ponderingdemocritus/SquireGPT#2]
![image](https://user-images.githubusercontent.com/57969364/229639574-071aef79-e31a-4ca2-8f5e-d383f9d8d9d9.png)

**Comment by ponderingdemocritus** [comment]
you running `yarn cli` and have your pinecone config set?

**Comment by vvonchain** [comment]
yeah; I have set the pinecone cli; for namespace I am giving the name of the index that I have created in pine cone for e.g "demo" and for index am giving this "demo-3d21f46.svc.us-east1-gcp.pinecone.io"
am I correct? I also tried swapping both; same error

**Comment by ponderingdemocritus** [comment]
ahh

Try it like this - you just need the name not the full string - ill update to make this clearer 

```
PINECONE_API_KEY=
PINECONE_INDEX="demo"
PINECONE_NAMESPACE="test"
```

**Comment by vvonchain** [comment]
did the above; getting the same error

**Comment by vvonchain** [comment]
this time; its just throwing failed to ingest the data
![image](https://user-images.githubusercontent.com/57969364/229782608-e232a182-0731-4a18-af78-fcff044bcac0.png)

**Comment by vvonchain** [comment]
its ingesting json files but not pdf
